File: eyan_helper/src/main/java/eu/eyan/testutil/ScalaEclipseJunitRunner.java
package eu.eyan.testutil;

import java.util.List;

import org.junit.runners.BlockJUnit4ClassRunner;
import org.junit.runners.model.InitializationError;

public class ScalaEclipseJunitRunner extends BlockJUnit4ClassRunner {

	public ScalaEclipseJunitRunner(Class<?> klass) throws InitializationError {
		super(klass);
	}

	@Override
	protected void validateTestMethods(List<Throwable> errors) {
	}

	@Override
	protected void validateInstanceMethods(List<Throwable> errors) {
	}

	@Override
	protected void collectInitializationErrors(List<Throwable> errors) {
	}
}

File: eyan_helper/src/main/scala/eu/eyan/testutil/ScalaEclipseJunitRunner.scala
package eu.eyan.testutil

import org.junit.runners.BlockJUnit4ClassRunner

class ScalaEclipseJunitRunner(klass: Class[_]) extends BlockJUnit4ClassRunner(klass) {
  override def validateTestMethods(errors: java.util.List[Throwable]) = {}
  override def validateInstanceMethods(errors: java.util.List[Throwable]) = {}
  override def collectInitializationErrors(errors: java.util.List[Throwable]) = {}
}